src/ui-components/QuickView.tsx

```
@@ -189,52 +194,78 @@ const QuickView = ({
+											style={theme == 'dark' ? { filter: 'invert(44%) sepia(98%) saturate(3203%) hue-rotate(309deg) brightness(101%) contrast(101%)' } : {}}
+											height={20}
+											width={20}
+											alt='edit logo'
+										/>
+										Tip{' '}
+									</div>
+								</CustomButton>
+							</div>
+						</Tooltip>
+					)}
+					{loginUserId && !isNaN(loginUserId) && (
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)